assets/i18n/active.de.json

```diff
@@ -25,39 +25,39 @@
   },
   "command.error.invalidInput": {
     "hash": "sha1-6ef073c836a398485967d7c10a085b611d272ddc",
-    "other": "Falsche Eingabe: {{.Error}}"
+    "other": "Ungültige Eingabe: {{.Error}}"
   },
   "command.error.invalidNumberOfOptions": {
     "hash": "sha1-31287872531872ed11e9ae69975afc1cd2ee09ae",
-    "other": "Du musst entweder keine Antwortmöglichkeit oder mindestens zwei angeben."
+    "other": "Du musst entweder keine Antwort oder mindestens zwei Antworten angeben."
   },
   "command.help.text.options": {
     "hash": "sha1-0b06ca6a7a2a1e2481ed761fe93eefec7d67079e",
-    "other": "Du kannst Antwortmöglichkeit festlegen, indem du `/{{.Trigger}} \"Frage\" \"Antwort 1\" \"Antwort 2\" \"Antwort 3\"` eingibst"
+    "other": "Du kannst die Optionen durch Eingabe von `/{{.Trigger}} \"Frage\" \"Antwort 1\" \"Antwort 2\" \"Antwort 3\"` anpassen"
   },
   "command.help.text.pollSetting.anonymous": {
     "hash": "sha1-413dccc620dcf0d910c2cba09522443793fbcb33",
-    "other": "Verberge, wer für was gestimmt hat"
+    "other": "Nach Abschluss der Umfrage nicht anzeigen, wer für was gestimmt hat"
```

Comment:
I supposed that this change adds to the meaning of "after the polls ends", but I don't want to limit `--anonymous` setting to after the polls ends. Because the ongoing PR (#431) may cause the A setting to also affect the anonymity of the ongoing poll.

Reply:
Yes, that's right, it means that. Of course, it also makes sense that anonymity is permanently preserved. But isn't that what the English text means? 

https://github.com/matterpoll/matterpoll/blob/master/server/plugin/command.go#L54

I believe this was translated the same way in most other translations. :thinking:

Reply:
Fixed

Reply:
Ah...yes, you're right. I was thinking too much about #431 and the English message had slipped my mind. 
Other languages including English also should be fixed.  But I plan to release #431 as next v1.6.0 release, so this message correction could have been made in the next v1.6.0 release. I'm sorry for confusing you, and either message is fine for this release.

Reply:
No problem, thanks for your work! Then we leave the text so and with the next release, it fits then?

Reply:
Sure 👍